docs/content/templates/functions.md

```diff
@@ -214,6 +214,14 @@ e.g.
     </ul>
 
 
+### intersect
+Given two arrays (or slices), this function will return a new set containing the elements out of both arrays. The elements supported are strings, integers and floats (only float64).
```

Comment:
I updated the description. What other general term would you use for an element of a set? Maybe  objects? The intersection function above refers to them as elements.